Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+P3A.swift

```diff
@@ -93,6 +95,75 @@ extension BrowserViewController {
     
     usage.recordReturningUsageMetric()
   }
+  
+  func recordAccessibilityDisplayZoomEnabledP3A() {
+    // Accessibility Q1 New P3A iOS - Do you have iOS display zoom enabled?
+    let isDisplayZoomEnabled = UIScreen.main.scale < UIScreen.main.nativeScale
+    
+    UmaHistogramBoolean("Brave.Accessibility.DisplayZoomEnabled", isDisplayZoomEnabled)
+  }
+  
+  func recordAccessibilityDocumentsDirectorySizeP3A() {
+    // Accessibility Q4 New P3A iOS - Documents directory size
+    func fetchDocumentsAndDataSize() -> Int? {
+      let fileManager = FileManager.default
+      
+      guard let documentsDirectory = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else {
+        return nil
+      }
+
+      do {
+        if let documentsDirectorySize = try fileManager.directorySize(at: documentsDirectory){
+          return Int(documentsDirectorySize / 1024 / 1024)
+        } else {
+          return nil
+        }
+      } catch {
+        return nil
+      }
+    }
+    
+    let buckets: [Bucket] = [
+      .r(0...50),
+      .r(50...200),
+      .r(200...500),
+      .r(500...1000),
+      .r(1000...),
+    ]
+    
+    if let documentsSize = fetchDocumentsAndDataSize() {
+      UmaHistogramRecordValueToBucket("Brave.Accessibility.DocumentDirectorySize", buckets: buckets, value: documentsSize)
+    }
+  }
+
+  func recordGeneralBottomBarLocationP3A() {
+    enum Answer: Int, CaseIterable {
+      case top = 0
+      case bottom = 1
+    }
+    
+    // General Q1 New P3A iOS - Bottom Bar Location
+    let answer: Answer = Preferences.General.isUsingBottomBar.value ? .bottom : .top
+    UmaHistogramEnumeration("Brave.General.BottomBarLocation", sample: answer)
+  }
+  
+  func recordTimeBasedNumberReaderModeUsedP3A(activated: Bool) {
```

Comment:
This is never called outside of activation, we may want to double check desktop's implementation of `Brave.SpeedReader.ToggleCount` and see if they report a 0 value when not used the entire week

Reply:
Looks like you are right, desktop is adding that case

https://github.com/brave/brave-core/blob/01a5b48c69489f28d1c4625c1f351deb0e01f0f7/components/speedreader/speedreader_service.cc#L51

Added launch record + enable/disable Fix at https://github.com/brave/brave-ios/pull/7100/commits/2696620bb9c7ae416c929338baff6ea432836c01